sim/midas/src/main/cc/bridges/bridge_driver.h

```diff
@@ -5,6 +5,14 @@
 
 #include "simif.h"
 
+namespace BridgeConstants {
+// Logically, Bridge streams are latency-insensitive FIFOs with a width of
+// this many bytes.
+// The host-implementation may use a different width under-the-hood but this
+// should not be exposed to bridge developers.
```

Comment:
Since you're using doxygen-style comments below, do you want this constexpr to be documented or are you intentionally hiding it?
```suggestion
/**
 *  @brief Logical byte width of Bridge streams
 * 
 * Bridge streams are logically latency-insensitive FIFOs with a width of
 * \c STREAM_WIDTH_BYTES.
 * \note { The host-implementation may use a different width under-the-hood but this
 * should not be exposed to bridge developers. }
 */
```

Reply:
No, i should've done that.